### ISSUE TITLE
O'FAIRe tool: Question A1Q3

### ISSUE THREAD

**Issue by martinaPulieri** [lifewatch-eric/ecoportal_web_ui#52]
Since now there is the SPARQL endpoint available, this question should always have 6 pt?

![image](https://github.com/lifewatch-eric/ecoportal_web_ui/assets/149166663/7023c1e6-d4da-42e7-9235-93ebdef299fb)

**Comment by jonquet** [comment]
This is fairly right. But this was the same befor ethis new upcoming release. In fact, this consume the metdata property sd:endpoint that is normally "automatically'" populated and could be curated after. 
But we did not choose to implement O'FAIRe considering the fact that the portal would always provide one... we left this behaviour at the level of the repository (metadata extractor + curation) ... 

In other words, the EcoPortal team needs to curate the sd:endpoint values
https://data.ecoportal.lifewatch.eu/submissions?display=endpoint

**Comment by martinaPulieri** [comment]
Okay, I thought that, since there was the content migration from the old version of EcoPortal (without the SPARQL endpoint) to this new one (with the SPARQL endpoint), the property sd:endpoint was populated. We will curate this then. Thanks for the clarification

**Comment by jonquet** [comment]
In fact this release will include a dedicated UI for the SPARQL endpoint (for each onto) see AgroPortal release notes: https://doc.jonquetlab.lirmm.fr/share/e6158eda-c109-4385-852c-51a42de9a412/doc/release-notes-btKjZk5tU2

But the SPARQL endpoint was technically alrady there in previous version and the metadata property was there too.
